packages/valaxy/node/utils/parseAddons.test.ts

```diff
@@ -0,0 +1,18 @@
+import { resolve } from 'path'
```

Comment:
I prefer to put the test files together like unocss/slidev.
Can you move it to the test folder?